src/Runtime/Context.php

```diff
@@ -0,0 +1,62 @@
+<?php declare(strict_types=1);
+
+namespace Bref\Runtime;
+
+class Context
+{
+    /** @var string */
+    private $awsRequestId = '';
+
+    /** @var int */
+    private $deadlineMs;
+
+    /** @var string */
+    private $invokedFunctionArn;
+
+    /** @var string */
+    private $traceId;
+
+    public function getAwsRequestId(): string
+    {
+        return $this->awsRequestId;
+    }
+
+    public function setAwsRequestId(string $awsRequestId): Context
+    {
+        $this->awsRequestId = $awsRequestId;
+        return $this;
+    }
+
+    public function getDeadlineMs(): int
```

Comment:
It looks like all documented context objects implement getRemainingTimeInMillis() instead of just providing the deadline timestamp. We should follow that.

Reply:
Strike that. All except Go of course.
Regardless, I did implement getRemainingTimeInMillis in the end as there is no "clean" way to get a milliseconds timestamp in PHP. I chose the simpler intval(microtime(true)*1000) and substract it from AWS's deadline.